src/argilla/client/sdk/v1/datasets/models.py

```diff
@@ -100,3 +100,19 @@ class FeedbackSuggestionModel(BaseModel):
     score: Optional[float] = None
     value: Any
     agent: Optional[str] = None
+
+
+class FeedbackRecordsMetricsModel(BaseModel):
+    count: int
+
+
+class FeedbackResponsesMetricsModel(BaseModel):
+    count: int
+    submitted: int
+    discarded: int
+    draft: int
```

Comment:
I think we don't need these metrics attributes in the python client for now. Or do we show the user connected metrics somewhere?

Reply:
It was just to align both schemas, but for now I guess we'll just use `records.count` from the Python client as part of `len(dataset.records)`